packages/mui-lab/src/DateRangePicker/DateRangePicker.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import * as React from 'react';
+
+let warnedOnce = false;
+
+const warn = () => {
```

Comment:
Looks great 👍 We should just revert the changes from `packages/mui-lab/src/index.js` and `packages/mui-lab/src/index.d.ts` (so that we still export them from `@mui/lab`).